### ISSUE TITLE
[Mod Manager] 2.2.1.34 - This mod will force the new Game's Game Settings to be in English.

### ISSUE THREAD

**Issue by Eamixs** [O-n-y/oxy.mods#147]
1. The OS you are running: Windows 10 LTSC 21H2 19044.1387
2. The games version number: U50-587362-SD
3. Mod Manager 2.2.1.34
4. mods enabled: only this Mod
![Снимок](https://github.com/O-n-y/oxy.mods/assets/55243637/8129ed1c-7acf-4f8e-acfc-bee16a726589)
![Снимок2](https://github.com/O-n-y/oxy.mods/assets/55243637/afeff7ee-9381-4a54-afff-feede86a2f22)

**Comment by MNWZ211** [comment]
I also have this problem. Whenever I only enable this mod (Mod Manager//by @ Ony), the built-in translation module in the game will fail (my game only settings will become the default language). This problem has been present for a long time, please fix it.
<img width="710" alt="image" src="https://github.com/O-n-y/oxy.mods/assets/80450278/4d474007-4da5-4f69-b42a-51f46ebe1da1">

**Comment by O-n-y** [comment]
do you still have this issue ?

**Comment by taozhiyu** [comment]
![image](https://github.com/user-attachments/assets/c39815dc-36c1-40ff-a9bc-37393e4c7fa4)
 Same.

**Comment by ripnghy** [comment]
Me too，so does the author plan to fix it?

**Comment by O-n-y** [comment]
no, it is required for it to be first in list (i.e. load first before other mods), to work properly.

**Comment by MNWZ211** [comment]
> 不，必须它排在列表第一（即在其他模组之前先加载），才能正常工作。

<img width="1919" height="984" alt="Image" src="https://github.com/user-attachments/assets/cc0c68da-9f6f-4301-9ae0-2152dd99dea4" />

<img width="1919" height="984" alt="Image" src="https://github.com/user-attachments/assets/59cefd60-dc8a-469e-8bf9-2c2b9ab08f5f" />
When only keeping this mod and not enabling other mods, the "Game Settings" panel when starting a new game will still be the default language, not the language I need. I have also tried closing several modules in the same way, suspecting that it is a problem with the main body.

**Comment by MNWZ211** [comment]
<img width="1919" height="984" alt="Image" src="https://github.com/user-attachments/assets/59d6e4ba-0388-44a4-a283-f3d6f82d7b2c" />

<img width="1919" height="984" alt="Image" src="https://github.com/user-attachments/assets/e2f8cc64-ddc2-4f28-b61b-544e9c558f32" />
I have also found that when the mod "No Drop" or "Mod Manager" is enabled separately, or when combined with other mods (only "Mod Manager" or "No Drop"), the translation of the game settings page in the new game will become invalid, which is not the language I need. When I disabled both modules, it returned to normal. As you said, I enabled 'Mod Manager' separately and placed it at the top of the list, but it still doesn't work.

**Comment by MNWZ211** [comment]
The module 'WGSM - World Generation Settings Manager' also experiences the same issue. Please fix this issue, and I also really like the mods you made. Your mods are very practical.
